packages/theme/src/cli/services/dev.ts

```diff
@@ -0,0 +1,66 @@
+import {renderConfirmationPrompt, renderSuccess, renderWarning} from '@shopify/cli-kit/node/ui'
+import {joinPath} from '@shopify/cli-kit/node/path'
+import {access} from 'node:fs/promises'
+
+export function renderLinks(store: string, themeId: string) {
+  renderSuccess({
+    body: [
+      {
+        list: {
+          title: 'Preview your theme',
+          items: [
+            {
+              link: {
+                url: 'http://127.0.0.1:9292',
```

Comment:
As users might pass `--host` and `--port` flags, we may extract both as parameters :)